dashboard/test/factories/factories.rb

```diff
@@ -161,6 +161,7 @@
   factory :section do
     sequence(:name) {|n| "Section #{n}"}
     user {create :teacher}
+    login_type 'email'
```

Comment:
Nope. My assumption is that most tests creating sections don't care about login_type, and if they do care about login_type they will specify one. I just wanted to provide any default.